README.md

```diff
@@ -89,7 +89,9 @@ sudo usermod -aG adm,audio,video,plugdev jibri
 ```
 
 ### Config files
-* Edit the `config.json` file (installed to `/etc/jitsi/jibri/config.json` by default) appropriately.
+* Edit the `application.conf` file (installed to `/etc/jitsi/jibri/application.conf` by default) appropriately.  You can look at
```

Comment:
Shoud we install to `/etc/jitsi/jibri/jibri.conf`?

Reply:
Think it needs to be named `jibri.conf` since it's in `/etc/jtisi/jibri` ?

Reply:
I find `application` confusing. The only reason we use it is because the undedrlying library refers to it this way. I think most services use similar naming, just scrolling through /etc on a random machine I find `/etc/dhcp/dhclient.conf`, `/etc/ldap/ldap.conf`, `/etc/lvm/lvm.conf`, `/etc/mdadm/mdadm.conf`, `/etc/nginx/nginx.conf`, `/etc/prosody/prosody.cfg.lua`, `/etc/ssl/openssl.cnf`, `/etc/ssh/sshd_config`. Contrast with `/etc/jitsi/videobridge/sip-communicator.properties` :)

Reply:
`application.conf` actually is a 'special' filename for the typesafe config library: it is loaded automatically as an override to `reference.conf`.  But, since we're using the `-Dconfig.file` argument instead of adding the config dir to the classpath, we can use whatever name we want.  Changed it to `jibri.conf`.